src/client/components/pages/entities/author-table.js

```diff
@@ -30,8 +30,8 @@ function AuthorTableRow({author, showAddedAtColumn, showCheckboxes, selectedEnti
 	const name = getEntityLabel(author);
 	const disambiguation = getEntityDisambiguation(author);
 	const number = author.number || '?';
-	const authorType = author.authorType ? author.authorType.label : '?';
-	const gender = author.gender ? author.gender.name : '?';
+	const authorType = author.authorType || author.authorType?.label || author?.type || '?';
```

Comment:
author.type here refers to the entity type rather than the author type (so it will always be "Author").
I know it's not very clear and some early choices of column names were not the smartest :p

Also, I would suggest this order:
```suggestion
	const authorType = author.authorType?.label || author.authorType || '?';
```